.github/CODE_STANDARDS.md

```diff
@@ -0,0 +1,295 @@
+# Code Standards & Contributing Guidelines
+
+## Table of Contents
+
+- [Code Standards \& Contributing Guidelines](#code-standards--contributing-guidelines)
+  - [Table of Contents](#table-of-contents)
```

Comment:
Unnecessary link

Reply:
You are right - it was automatically created by my VSCode extension, sorry.